lib/lte_link_control/lte_lc.c

```diff
@@ -661,7 +661,10 @@ static int init_and_config(void)
 static int connect_lte(bool blocking)
 {
 	int err;
-	bool retry;
+	int tries;
+	enum lte_lc_func_mode current_func_mode;
+
+	tries = (IS_ENABLED(CONFIG_LTE_NETWORK_USE_FALLBACK) ? 2 : 1);
```

Comment:
This can be assigned right away at line 664

Reply:
Sounds good. Updated!